### PR TITLE
Added GitHub Pages

### PR DESCRIPTION
**Pull request by Agustin-Mediotti**

### CODE REVIEW

**Review by adrianovi — DISMISSED** [review]

**Review by eliosfg — CHANGES_REQUESTED** [review]

File: .github/workflows/github-actions-pipeline.yml [hunk 1]
```diff
@@ -64,21 +68,19 @@ jobs:
...
-#          PUBLISH_DIR: "reports/cucumber-html-reports/overview-features.html"
\ No newline at end of file
+          path: "build/cucumber/*.json"
+      - name: Deploy report to Github Pages
+        if: always()
+        uses: peaceiris/actions-gh-pages@v2
+        env:
+          PERSONAL_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PUBLISH_BRANCH: gh-pages
+          PUBLISH_DIR: "reports/cucumber-html-reports/overview-features.html"
```
[the remark this answers is not in the document]
Comment (Agustin-Mediotti) [reply .github/workflows/github-actions-pipeline.yml]: Changed to requested directory

**Review by jimytz — APPROVED** [review]
👍

**Review by adrianovi — DISMISSED** [review]

**Review by eliosfg — APPROVED** [review]
Good

**Review by adrianovi — DISMISSED** [review]

**Review by SerJimmy — DISMISSED** [review]

**Review by jimytz — APPROVED** [review]
💯

**Review by adrianovi — APPROVED** [review]

**Pull request merged**